pkg/providers/ingress/ingress.go

```diff
@@ -428,7 +412,15 @@ func (c *ingressController) ResourceSync() {
 		if !c.namespaceProvider.IsWatchingNamespace(key) {
 			continue
 		}
+
 		ing := kube.MustNewIngress(obj)
+		if !c.isIngressEffective(ing) {
+			log.Debugw("ignore noneffective ingress add event",
```

Comment:
Only the log of the sync object should be printed, and the sync should not care about the ingresssclass log.

Reply:
I misunderstanding sonething. And i will modify it soon